### DISCUSSION TITLE
is there a way to add localsend into windows right click menu so that i can right click sth and send it via localsend

### DISCUSSION THREAD

**Discussion by CRHcs** [localsend/localsend#1184] [Discussion (English, International)]
is there a way to add localsend into windows right click menu so that i can right click sth and send it via localsend

**Comment by Tienisto** [comment]
It is not implemented yet. See #115

**Comment by junceo** [comment]
Mac中也很需要这个功能，希望大佬增加该功能，幸苦啦
